client/app/src/main/java/io/mosip/registration/app/activites/JobServiceActivity.java

```diff
@@ -129,27 +125,33 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
                 if (isChecked) {
                     try {
                         int resultCode = jobServiceHelper.scheduleJob(jobServiceModel.getId(), jobServiceModel.getApiName());
-
                         if (resultCode == JobScheduler.RESULT_SUCCESS) {
                             Log.d(TAG, "Job scheduled");
+                            jobServiceModel.setEnabled(true);
+                            jobServiceModel.setLastSyncTime(jobServiceHelper.getLastSyncTime(jobServiceModel.getId()));
                             Toast.makeText(JobServiceActivity.this, "Job scheduled", Toast.LENGTH_SHORT).show();
```

Comment:
Add these constants in the strings.xml under values to support internationalization.

Reply:
Done